### ISSUE TITLE
Did you find a way to have /dev/net/tun (tun module) in Debian Candyhouse kernel

### ISSUE THREAD

**Issue by cilynx** [cilynx/Candyhouse-Linux#4]
http://www.wolfteck.com/projects/candyhouse/install/#comment-1975829450

**Comment by bashou** [comment]
Hi here !

It won't work anymore ... maybe it's my usb stick who need to be "reinstalled".
It worked with 3.13.7, what do you think ?

Thanks.

**Comment by cilynx** [comment]
The image that I built at the airport didn't work because CONFIG_CMDLINE got reverted to looking for the rootfs in NAND instead of on USB.  I fixed that for this one and confirmed that it boots on my EA4500.  What symptoms are you seeing when it doesn't boot?  Have you poked through /var/log/[dmesg, kern.log, messages, syslog] to see if there's any hint there?  If none of the logs change (use checksums, timestamps aren't reliable during boot on this platform), then it might not be finding your USB.  Does your USB stick's activity light flash during the failed boot attempts?

root@candyhouse:~# cat /proc/version 
Linux version 3.19.5-candyhouse-openwrt (rcw@initiative) (gcc version 4.8.4 20141219 (release) (4.8.4-1+11-1) ) Thu Apr 23 12:08:11 PDT 2015
